src/todos.js

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable no-use-before-define */
```

Comment:
- [Optional] It's not a good approach to disable linters, fixing linters help us to write a clean, clear, and good codebase, kindly make sure to not disable linters 🙏🏾